builtin/providers/google/config.go

```diff
@@ -36,6 +36,13 @@ type Config struct {
 
 func (c *Config) loadAndValidate() error {
 	var account accountFile
+	clientScopes := []string{
+		"https://www.googleapis.com/auth/compute",
+		"https://www.googleapis.com/auth/cloud-platform",
```

Comment:
In principle it should be possible to have only cloud-platform as all the others are meant to be subsumed by cloud-platform.  When all PRs are in, you could try running the acceptance tests with just cloud-platform and see if it works.